pkg/orm/dojo_worker.go

```diff
@@ -33,6 +77,18 @@ func (s *DojoWorkerORM) CreateDojoWorker(walletAddress string, chainId string) (
 }
 
 func (s *DojoWorkerORM) GetDojoWorkerByWalletAddress(walletAddress string) (*db.DojoWorkerModel, error) {
+	workerCache := NewDojoWorkerCache(WorkerByWalletCacheKey)
+	workerCache.walletAddress = walletAddress
+
+	var worker *db.DojoWorkerModel
+	cache := cache.GetCacheInstance()
+
+	// Try to get from cache first
+	if err := cache.GetCache(workerCache, &worker); err == nil {
```

Comment:
this line could be clearer if you had `key` as the first param. alternatively, you can rename `workerCache` to `workerCacheParams`